x-pack/plugins/beats_management/server/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { beatsManagementConfigSchema } from '../common';
+
+export const config = {
+  schema: beatsManagementConfigSchema,
+
+  exposeToBrowser: {
+    defaultUserRoles: true,
+    encryptionKey: true,
+    enrollmentTokensTtlInSeconds: true,
+  },
+};
+
+export const plugin = () => ({
```

Comment:
Is it necessary to have a server plugin defined at this point? I noticed server is enabled in `kibana.json`, just curious since we aren't migrating server in this patch. Basically I'm wondering what this code is going to do in the meantime.

Reply:
It is necessary to be able to define the config schema.